tutorials/deploy-xenforo-to-compute-engine/index.md

```diff
@@ -0,0 +1,267 @@
+
+Instructions below are general procedures to set up a mailing service for this tutorial:
+
+1. Sign up with the mailing service you would like to use. Write down the SMTP username and the SMTP password provided by the service.
+
+2. Connect to your Compute Engine instance and change to the directory of postfix preferences:
+
+	```bash
+	cd /etc/postfix
+	```
+
+3. Edit the preferences:
+
+	```bash
+	nano main.cf
+	```
+
+	Use arrow keys (UP, DOWN, LEFT and RIGHT keys) to move the cursor around. Comment out the following lines:
+
+	| Before                    | After                     |
+	| --------------------------|---------------------------|
+	| default_transport = error | #default_transpot = error |
+	| relay_transport = error   | #relay_transport = error  |
+
+	Edit the following lines:
+
+	| Before      | After                             |
+	| ------------|-----------------------------------|
+	| relayhost = | relayhost = [RELAY_SERVER]:[PORT] |
+
+	where the `[RELAY_SERVER]` and `[PORT]` are provided by the mailing service. Note that you cannot use common SMTP ports such as 25, 465 or 587. If you are using SendGrid, Mailgun or Mailjet, here are their relay services and ports for Compute Engine instances, as seen in the [Compute Engine documentation](https://cloud.google.com/compute/docs/tutorials/sending-mail/):
+
+	| Mailing Service | Relay Server      | Port |
+	| --------------- |-------------------|------|
+	| SendGrid        | smtp.sendgrid.net | 2525 |
+	| Mailgun         | smtp.mailgun.org  | 2525 |
+	| Mailjet         | in.mailjet.com    | 2525 |
+
+	and finally, move to the end of the file and add the following lines:
+
+	```
+	smtp_tls_security_level = encrypt
+	smtp_sasl_auth_enable = yes
+	smtp_sasl_password_maps = hash:/etc/postfix/sasl_passwd
+	smtp_sasl_security_options = noanonymous
+	```
+
+4. Generate a new SASL password map and convert it to a .db file:
+
+	```bash
+	sudo nano sasl_password
+	```
+
+	Enter this line:
+
+	```
+	[RELAY_SERVER]:[PORT] [SMTP_USERNAME]:[SMTP_PASSWORD]
+	```
+
+	Replace `[RELAY_SERVER]`, `[PORT]`, `[SMTP_USERNAME]` and `[SMTP_PASSWORD]` with the values you used or recorded earlier.
+
+	Press Ctrl + O (Control Key and O Key together) to write the file. Then press Ctrl + X (Control Key and X Key together) to exit.
+
+	Convert the file you just created to a .db file, grant proper access permission to it and remove the old file with
+
+	```bash
+	sudo postmap sasl_password
+	sudo chmod 600 sasl_password.db
+	sudo rm sasl_password
+	```
+
+5. Lastly, restart the postfix service:
+
+	```bash
+	sudo service postfix restart
+	```
+
+	And now your XenForo website can use the third-party service you choose to send emails more securely and efficiently.
```

Comment:
"Now your XenForo website can use the third-party service to send emails more securely and efficiently."